### ISSUE TITLE
Framework Delegate is Missing Zone <root> (ion-nav root component not showing in browser)

### ISSUE THREAD

**Issue by Simbaclaws** [zyra/ionic-super-tabs#456]
Hi there,

I'm using ionic 5 and I'm trying to use the ion-super-tabs component.
The following html is what I'm using for the ion-super-tabs:
```
<super-tabs [config]="tabOpties">
  <super-tabs-toolbar slot="top" color="primary">
    <super-tab-button>
      <ion-icon name="chatbox"></ion-icon>
      <ion-label>Nieuws</ion-label>
    </super-tab-button>
    <super-tab-button #twitterTab>
      <ion-icon name="logo-twitter"></ion-icon>
      <ion-label>Twitter</ion-label>
    </super-tab-button>
    <super-tab-button #facebookTab>
      <ion-icon name="logo-facebook"></ion-icon>
      <ion-label>Facebook</ion-label>
    </super-tab-button>
  </super-tabs-toolbar>

  <super-tabs-container>
    <super-tab>
      <ion-nav [root]="nieuws" #nieuwsNav></ion-nav>
    </super-tab>
    <super-tab #twitterPage>
      <ion-nav [root]="twitter" #twitterNav></ion-nav>
    </super-tab>
    <super-tab #facebookPage>
      <ion-nav [root]="facebook" #facebookNav></ion-nav>
    </super-tab>
  </super-tabs-container>
</super-tabs>
```
This seems to render the pages on a mobile device (native android), but not inside of the web browser (chrome and safari).
I'm setting the variables nieuws, twitter and facebook to the corresponding pages imported in the ts file like so:
```
import { FacebookPage } from './facebook/facebook.page';
import { TwitterPage } from './twitter/twitter.page';
import { NieuwsPage } from './nieuws/nieuws.page';
import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
@Component({
  selector: 'app-berichten',
  templateUrl: './berichten.page.html',
  styleUrls: ['./berichten.page.scss'],
})
export class BerichtenPage implements OnInit {
  nieuws = NieuwsPage;
  twitter = TwitterPage;
  facebook = FacebookPage;
  tabOpties = {
    sideMenu: 'left',
    allowElementScroll: true,
    debug: true
  };
  constructor(
    private weergave: WeergaveService
  ) { }

  ngOnInit() {

  }
}
```

The error message I'm getting in the console is as followed:
```
zone.js:690 Unhandled Promise rejection: framework delegate is missing ; Zone: <root> ; Task: Promise.then ; Value: Error: framework delegate is missing
```

Am I missing something here?
I have SuperTabsModule imported in both my berichten.module.ts and in app.module.ts where in app.module.ts I set the imports to SuperTabsModule.forRoot() and in berichten.module.ts I just have SuperTabsModule imported.

**Comment by Simbaclaws** [comment]
Hmmm apparently this console message only appears when running in development mode.
When I do ionic serve --prod, the pages are displayed and the console message is gone.

I would like to find a solution for this, but for now I'll run with --prod mode on a local machine to do tests.

**Comment by kunanfo** [comment]
Yes I am also face this problem.  And once I run with --prod, there is some error : Error: src/app/main/cms/cms.page.html:25:16 - error NG8002: Can't bind to 'root' since it isn't a known property of 'ion-nav'.
1. If 'ion-nav' is an Angular component and it has 'root' input, then verify that it is part of this module.
[ng] 2. If 'ion-nav' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.

[INFO] Development server running!
       
       Local: http://localhost:8100
       
       Use Ctrl+C to quit this process

[INFO] Browser window opened to http://localhost:8100!

[ng] 3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component.
[ng] 25       <ion-nav [root]="newsPage"></ion-nav>
[ng]                   ~~~~~~~~~~~~~~~~~
[ng]   src/app/main/cms/cms.page.ts:6:16
[ng]     6   templateUrl: './cms.page.html',
[ng]                      ~~~~~~~~~~~~~~~~~


Ionic:

   Ionic CLI                     : 6.16.2 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.8.0
   @angular-devkit/build-angular : 12.1.4
   @angular-devkit/schematics    : 12.1.4
   @angular/cli                  : 12.1.4
   @ionic/angular-toolkit        : 4.0.0

Capacitor:

   Capacitor CLI      : 3.2.3
   @capacitor/android : not installed
   @capacitor/core    : 3.2.3
   @capacitor/ios     : 3.2.3

Utility:

   cordova-res (update available: 0.15.3) : 0.15.2
   native-run                             : 1.4.1

System:

   NodeJS : v14.17.5 (/usr/local/bin/node)
   npm    : 6.14.14
   OS     : macOS Big Sur

**Comment by marfrede** [comment]
I get the same error now and then with ionic@6
